### DISCUSSION TITLE
How to access Strimzi Kafka exporter outside Strimzi namespace

### DISCUSSION THREAD

**Discussion by iamrsaravana** [strimzi/strimzi-kafka-operator#5335] [Q&A]
Based on  Strimzi documentation, we are able to install Kafka and Prometheus in same name space and working fine. but we are planning to have common Prometheus to collect all my applications related metrics so install Prometheus in in "Prometheus namespace" and pull data from Kafka exporter from strimzi. 

Could you tell us where to assign external  IP address and port number for Kafka exporter.

**Comment by scholzj** [comment]
I do not think you need anything special for it - no external IP address or port number. You just need to tell your Prometheus to scrape also from other namespaces. I do not know how you deployed your Prometheus. All I needed to do in my deployment is to configure the right namespace in the PodMonitor resource used by the Prometheus Operator.
